discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/ListConfigurationsResponse.java

```diff
@@ -10,23 +10,35 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-
-package com.ibm.watson.developer_cloud.discovery.v1.model.collection.field;
+package com.ibm.watson.developer_cloud.discovery.v1.model;
 
 import java.util.List;
 
-import com.google.gson.annotations.SerializedName;
-import com.ibm.watson.developer_cloud.discovery.v1.model.collection.CollectionManager;
 import com.ibm.watson.developer_cloud.service.model.GenericModel;
 
 /**
- * Response from {@link GetCollectionFieldsRequest}.
+ * ListConfigurationsResponse.
  */
-public class GetCollectionFieldsResponse extends GenericModel {
-    @SerializedName(CollectionManager.FIELDS)
-    private List<Field> fields;
+public class ListConfigurationsResponse extends GenericModel {
+
+  /** An array of Configurations that are available for the service instance. */
+  private List<Configuration> configurations;
+
+  /**
+   * Gets the configurations.
+   *
+   * @return the configurations
+   */
+  public List<Configuration> getConfigurations() {
+    return configurations;
+  }
 
-    public List<Field> getFields() {
-        return fields;
-    }
+  /**
+   * Sets the configurations.
+   *
+   * @param configurations the new configurations
+   */
+  public void setConfigurations(final List<Configuration> configurations) {
```

Comment:
This one is clearly wrong. But I'm not sure we can make the blanket decision to eliminate setters on all response objects.  Need to think about this one.